### PR TITLE
#146: Removed public modifier from most of the methods

### PR DESCRIPTION
**Pull request by bonkey**
- Reduced access from public to internal:
  - UserMessage
  - NSURLComponents
  - CheckoutViewModel
  - Async
  - UIImage(named:bundledWith:)
  - UIImageView.setImage (and added AlamofireImage to AtlasDemo instead)
- Dropped UIConstants
- Duplicated code from UserMessage in AtlasDemo as UIViewController extension

### CODE REVIEW

**Comment by bonkey** [comment]
👍

File: AtlasDemo/AtlasDemo/UIViewController.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        init(text: String, handler: ButtonActionHandler? = nil) {
+            self.text = text
+            self.handler = handler
+        }
+
+    }
+
+    func showOK(title title: String, message: String) {
+        showMessage(title: title, message: message, actions: ButtonAction(text: "OK"))
```
Comment (hani-ibrahim) [inline AtlasDemo/AtlasDemo/UIViewController.swift]: localize `OK` ?
same in `showError` function

Comment (hani-ibrahim) [inline AtlasDemo/AtlasDemo/UIViewController.swift]: Aha it's the demo, okay never mind 👍

Comment (bonkey) [inline AtlasDemo/AtlasDemo/UIViewController.swift]: Well, this could be localized in `AtlasUI` actually.

**Comment by hani-ibrahim** [comment]
👍

**Comment by bonkey** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Pull request merged**